README.md

```diff
@@ -212,6 +219,34 @@ After the Flexy Java application component has been installed and started, the c
 ### Thingworx Base URL 
 This parameter in the config file should be changed to the base URL for the target Thingworx REST API. Users should pay close attention to the scheme designated in the URL. Specifying the scheme "https" will ensure all traffic to the server is encrypted over TLS. For example, setting the ThingworxBaseUrl parameter to "https://example.com:8443" will create a connection to host "example.com" on port 8443 using TLS/SSL. It's possible to only specify the IP address of the Thingworx instance. This configuration will assume "http" scheme and communication will not be encrypted. 
 
+After the Flexy Java application component has been installed and started, the configuration should be modified to contain the proper values for AppKey and Thingworx Full URL for your Thingworx instance.
+
+![Configuration Example](https://github.com/hms-networks/flexy-thingworx-connector/blob/main/images/ExampleConfig.PNG?raw=true)
+
+#### Thingworx Full URL
+This parameter in the config file should be changed to the full URL for the target Thingworx REST API. Users should pay close attention to the scheme designated in the URL. Specifying the scheme "https" will ensure all traffic to the server is encrypted over TLS. For example, setting the ThingworxFullUrl parameter to "https://example.com:8443/Thingworx/Things/ConnectorHost/Services/TakeInfo" will create a connection to host "example.com" on port 8443 using TLS/SSL. It's possible to specify the IP address of the Thingworx instance instead of a domain name. This configuration will assume "http" scheme and communication will not be encrypted.
+
+#### App Key
+This should match the app key for the Thingworx instance.
+
+#### Log Level
+The Thingworx connector uses the HMS Solution Center logging library for application logging to the Ewon Flexy's realtime logs. See [Log Level](https://github.com/hms-networks/sc-flexy-logger-lib#log-level) for more information.
+
+#### FTP Username
+The username for accessing Ewon Flexy via FTP. This user account must be configured for non UTC time zones to be used on the Flexy.
+
+#### FTP Password
+The password for accessing Ewon Flexy via FTP. This user account must be configured for non UTC time zones to be used on the Flexy.
+
+#### Queue Enable String History
```

Comment:
I think we should include the default values for the added settings since those will be used if not explicitly set by the user. Knowing what the default value is may change whether or not they want to override it, and its easier than making them look in the source code for the value.